compiler/testSuite/Vbooleanorder01.bal

```diff
@@ -0,0 +1,11 @@
+import ballerina/io;
```

Comment:
What does this add to Vbooleanorder00?

Reply:
directly comparing the boolean values without assigning is tested here for code coverage